pastas/stressmodels.py

```diff
@@ -47,8 +51,7 @@ class StressModelBase:
     """
     _name = "StressModelBase"
 
-    def __init__(self, name, tmin, tmax, rfunc=None, up=True, meanstress=1,
-                 cutoff=0.999):
+    def __init__(self, name: str, tmin: Tminmax, tmax: Tminmax, rfunc: Optional[RFunc] = None, up: bool = True, meanstress: float = 1.0, cutoff: float = 0.999) -> None:
```

Comment:
line too long

Reply:
formatted all functions and methods :)))